opensearch/opensearch.gemspec

```diff
@@ -29,7 +29,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'opensearch/version'
 
 Gem::Specification.new do |s|
-  s.name          = 'opensearch'
+  s.name          = 'opensearch-ruby'
```

Comment:
@dblock You views on this change ?
So we are proposing changing only the name of the Gem to be changed  (from `opensearch` to `opensearch-ruby` ) and not the folder structure anywhere.

Reply:
I think it's fine.